packages/react/package.json

```
@@ -28,9 +28,11 @@
 	"homepage": "https://github.com/IBM/carbon-charts#readme",
 	"dependencies": {
 		"@carbon/charts": "^0.11.0",
```

meant to include this in the last review, oops.
```suggestion
		"@carbon/charts": "^0.11.0"
```